st2reactor/st2reactor/container/sensor_wrapper.py

```diff
@@ -276,8 +277,11 @@ def run(self):
         try:
             self._sensor_instance.run()
         except Exception as e:
-            raise Exception('Sensor "%s" run method raised an exception: %s' %
-                            (self._class_name, str(e)))
+            # Include traceback
+            msg = 'Sensor "%s" run method raised an exception: %s.\nTraceback: %s'
+            msg = msg % (self._class_name, str(e), traceback.format_exc())
+            self._logger.warn(msg)
```

Comment:
I guess you do it to include some more information. Generally, the double traceback server minimal value especially if the exception is logged in the same file as the original. 

Anyway its done. So be it.